index.html

```diff
@@ -4,9 +4,26 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <title>Oshane Design Portfolio</title>
 </head>
-<body>
-    <h1>Text</h1>
+<body> 
+    <div class="main-Container"> 
+    <header class="navBar">
+        <p class="my-logo">My Logo</p>
+        <img class="menuDropDown" src="images/Union.svg" alt="menuDropDown">
+    </header>
+    <section class="section1">
+        <div class="img-background"> 
+        <h1>I'm Mariane <br> Glad to see <br> you!</h1>
+         <p>I am a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, do not hestiate to contact me.</p>
+        <h2>LET'S CONNECT</h2>
+            <ul> 
+                <li><img class="section1-Icons" src="images/social-media-icons.svg" alt="social-Media-Icons"></li>
+            </ul>  
+        </div>
+    </section>
```

Comment:
- [ ] You have done great work, but kindly align the contents of this section using `flexbox`, please, let's go and fix that 🏃🏻  😄

Reply:
thank you, fixed.@BenMukebo

Reply:
Great fix!